### PR TITLE
Remove disk file in DataBlock's deconstructor

### PR DESCRIPTION
**Pull request by yvxiang**

### CODE REVIEW

**Comment by lylei** [comment]
what about some unit-tests?

**Comment by yvxiang** [comment]
inprogress

**Comment by yvxiang** [comment]
build

File: src/chunkserver/data_block.cc [hunk 1]
```diff
@@ -114,6 +114,19 @@ Block::~Block() {
         }
         delete recv_window_;
     }
+
+    if (deleted_) {
+        int64_t du = DiskUsed();
+        int ret = remove(disk_file_.c_str());
+        if (ret != 0 && (errno !=2 || du > 0)) {
```
Comment (lylei) [inline src/chunkserver/data_block.cc]: 用宏名字好读一点

Comment (yvxiang) [inline src/chunkserver/data_block.cc]: done

**Pull request merged**